src/app/containers/TodoApp/index.tsx

```diff
@@ -187,24 +201,29 @@ class TodoApp extends React.Component<Props, State> {
                 />
                 <label className={style.todoText}>{todo.id}</label>
                 <label className={style.todoText}>{todo.text}</label>
+                <button type="button" className={style.editButton} onClick={() => this.openModalForEdit(todo.id)}>
+                  edit Todo
```

Comment:
```suggestion
                  edit
```
で良いと思います。
よろしければ、このUI上でコミットして、pullしてみましょう。